exporter/elasticsearchexporter/data_stream_router.go

```diff
@@ -38,6 +40,13 @@ func routeWithDefaults(defaultDSType, defaultDSDataset, defaultDSNamespace strin
 				return fmt.Sprintf("%s%s%s", prefix, fIndex, suffix)
 			}
 		}
+
+		// The naming convention for datastream is expected to be the following
```

Comment:
maybe mention that this is so that (soon to be) built-in `logs-*.otel-*` index template is matching

Reply:
Done